huaweicloud/services/mrs/data_source_huaweicloud_mapreduce_clusters.go

```diff
@@ -0,0 +1,681 @@
+			},
+			"stage_desc": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Cluster progress description.`,
+			},
+			"is_mrs_manager_finish": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: `Whether MRS Manager installation is finished during cluster creation.`,
+			},
+			"safe_mode": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Running mode of an MRS cluster.`,
+			},
+			"version": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Cluster version.`,
+			},
+			"node_public_cert_name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Name of the key file.`,
+			},
+			"master_node_ip": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `IP address of a Master node.`,
+			},
+			"private_ip_first": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Preferred private IP address.`,
+			},
+			"tags": {
+				Type:     schema.TypeMap,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"log_collection": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Whether to collect logs when cluster installation fails.`,
+			},
+			"task_node_groups": {
+				Type:        schema.TypeList,
+				Elem:        mrsClustersClustersNodeGroupSchema(),
+				Computed:    true,
+				Description: `List of Task nodes.`,
+			},
+			"node_groups": {
+				Type:        schema.TypeList,
+				Elem:        mrsClustersClustersNodeGroupSchema(),
+				Computed:    true,
+				Description: `List of Master, Core and Task nodes.`,
+			},
+			"master_data_volume_type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Data disk storage type of the Master node.`,
+			},
+			"master_data_volume_size": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Data disk storage space of the Master node`,
+			},
+			"master_data_volume_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Number of data disks of the Master node`,
+			},
+			"core_data_volume_type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Data disk storage type of the Core node.`,
+			},
+			"core_data_volume_size": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Data disk storage space of the Core node.`,
+			},
+			"core_data_volume_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Number of data disks of the Core node.`,
+			},
+			"period_type": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Whether the subscription type is yearly or monthly.`,
+			},
+			"scale": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Status of node changes.`,
+			},
+			"eip_id": {
```

Comment:
what's the difference between `eip_id` and `external_ip`?

Reply:
external_ip is exist always